css/amp.css

```diff
@@ -346,16 +346,15 @@ i-amp-scroll-container.amp-active {
 .i-amphtml-loader-moving-line {
   display: block;
   position: absolute;
-  left: -100%;
```

Comment:
I think it did. https://github.com/ampproject/amphtml/blob/master/css/amp.css#L361